src/Sulu/Component/SmartContent/ContentType.php

```diff
@@ -279,7 +279,7 @@ public function getContentData(PropertyInterface $property)
             $filters['limitResult'] : null;
         $options = [
             'webspaceKey' => $this->requestAnalyzer->getWebspace()->getKey(),
-            'locale' => $property->getStructure()->getLanguageCode(),
+            'locale' => $property->getStructure()->getDocument()->getOriginalLocale() ?? $property->getStructure()->getLanguageCode(),
```

Comment:
@sulu/core-team we should discuss this behaviour. I think currently `Smart Content` and a lot of `Selections` (Pages, Snippets, ...). are targeting the false local and eventually load content which is not available in the language I'm currently loading.

In @martinlagler specific case they are on a page `zh` which load `en` via shadow feature `en` = `$property->getStructure()->getLanguageCode()` but when clicked on an `Article` a `404` is shown as the rendered url `/zh/some-english-url` the article itself does not exist.

Reply:
@martinlagler managed that this solution currently run in the project where it was used into other errors where first a deeper dive into this topic is required why there 500 errors did appear then.

Reply:
There was also an inconsistent between pages `getLanguageCode` vs article `getLanguageCode` which could end into this kind of error.

This was fixed by @wachterjohannes in: https://github.com/sulu/SuluArticleBundle/pull/644

So maybe this PR get now obsolete.

Reply:
@alexander-schranz yes that could be possible - if it only happens for articles! if not we have to takkle that

Reply:
@Prokyonn you did have a look as I remember at the shadow implementation. Can you confirm that this issue was already fixed? On language `zh` which was a shadow to `en` all `en` smart items where returned, instead of the `zh` available items.

See comment at: https://github.com/sulu/sulu/pull/7091#discussion_r1221792048